src/mpyl/project.py

```diff
@@ -588,6 +598,21 @@ def validate_project(yaml_values: dict) -> dict:
     return yaml_values
 
 
+def load_possible_parent(
+    root_dir: Path,
+    project_path: Path,
+    safe: bool = False,
+) -> Optional[dict]:
+    full_path = root_dir / project_path
+    if str(full_path).endswith(Project.project_yaml_path()):
+        return None
+    parent_project_path = full_path.parents[1] / Project.project_yaml_path()
+    if not parent_project_path.exists():
+        return None
+    with open(parent_project_path, encoding="utf-8") as file:
+        return YAML(typ=None if safe else "unsafe").load(file)
```

Comment:
```suggestion
def load_possible_parent(
    full_path: Path,
    safe: bool = False,
) -> Optional[dict]:
    parent_project_path = full_path.parents[1] / Project.project_yaml_path()
    if (
        os.path.basename(full_path) == Project.project_yaml_path()
        or not parent_project_path.exists()
    ):
        return None
    with open(parent_project_path, encoding="utf-8") as file:
        return YAML(typ=None if safe else "unsafe").load(file)
```

Reply:
https://github.com/Vandebron/mpyl/pull/287/commits/91bb7489d9cfdacc6f7855d6af138c8025478c99

Reply:
that's missing the part of my suggestion where the `return None` is combined though

Reply:
the `basename` can not be used here
The two if blocks are merged here:
https://github.com/Vandebron/mpyl/pull/287/commits/7e7a3c7ec43c20dfc33b926466c73d79d02f17ea